### PR TITLE
Fix/support null values

### LINKED ISSUE #22: NPE when exporting an EXASOL table with NULL values
When exporting an EXASOL table that contains NULL values like this:

    EXPORT
    myschema.mytable
    INTO SCRIPT myschema.EXPORT_HCAT_TABLE WITH
    HCAT_DB = 'mydb'
    HCAT_TABLE = 'mytable'
    HCAT_ADDRESS = 'thrift://hcat01:9083,thrift://hcat02:9083'
    HDFS_URL = 'hdfs://hdfsnn01:8020,hdfs://hdfsnn02:8020'
    HDFS_USER = 'hdfs/_HOST@MYREALM.EXAMPLE.COM'
    AUTH_TYPE = 'kerberos'
    AUTH_KERBEROS_CONNECTION = 'myconnection'
    ;

This results in the following NPE:

    [22002] VM error: 
    java.lang.NullPointerException
    Stack trace:
    parquet.io.api.Binary$FromStringBinary.encodeUTF8(Binary.java:209)
    parquet.io.api.Binary$FromStringBinary.<init>(Binary.java:204)
    parquet.io.api.Binary.fromString(Binary.java:477)
    com.exasol.hadoop.parquet.Tuple.writePrimitiveValue(Tuple.java:97)
    com.exasol.hadoop.parquet.TupleWriter.writeTuple(TupleWriter.java:29)
    com.exasol.hadoop.parquet.TupleWriter.write(TupleWriter.java:19)
    com.exasol.hadoop.parquet.TupleWriteSupport.write(TupleWriteSupport.java:36)
    com.exasol.hadoop.parquet.TupleWriteSupport.write(TupleWriteSupport.java:11)
    parquet.hadoop.InternalParquetRecordWriter.write(InternalParquetRecordWriter.java:116)
    parquet.hadoop.ParquetWriter.write(ParquetWriter.java:324)
    com.exasol.hadoop.HdfsSerDeExportService$1.run(HdfsSerDeExportService.java:94)
    com.exasol.hadoop.HdfsSerDeExportService$1.run(HdfsSerDeExportService.java:71)
    java.security.AccessController.doPrivileged(Native Method)
    javax.security.auth.Subject.doAs(Subject.java:422)
    org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1920)
    com.exasol.hadoop.HdfsSerDeExportService.exportToParquetTable(HdfsSerDeExportService.java:71)
    com.exasol.hadoop.scriptclasses.ExportIntoHiveTable.run(ExportIntoHiveTable.java:155)

### PR DESCRIPTION
**Pull request by ujuettner**
Fixes #22 incl. tests.

### CODE REVIEW

File: src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java [hunk 1]
```diff
@@ -205,13 +254,25 @@ public void testExportParquetString() throws Exception {
         columns.add(new HCatTableColumn("v1", "varchar(1)"));
         columns.add(new HCatTableColumn("v2", "varchar(10)"));
         columns.add(new HCatTableColumn("s1", "string"));
+        columns.add(new HCatTableColumn("charnull", "string"));
```
Comment (exazg) [inline src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java]: The dataType should probably be char(X).

File: src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java [hunk 2]
```diff
@@ -205,13 +254,25 @@ public void testExportParquetString() throws Exception {
...
+        columns.add(new HCatTableColumn("varcharnulll", "string"));
```
Comment (exazg) [inline src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java]: The dataType should probably be varchar(X).

File: src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java (hunk 1 above)
Comment (ujuettner) [reply src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java]: Sorry, my bad - copy&paste error. Will fix this.

File: src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java (hunk 2 above)
Comment (ujuettner) [reply src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java]: Sorry, my bad - copy&paste error. Will fix this.

**Comment by exazg** [comment]
Thanks again for the fixes and tests!

**Pull request merged**